src/python/dxpy/bindings/dxfile_functions.py

```diff
@@ -179,6 +179,22 @@ def _download_symbolic_link(dxid, md5digest, project, dest_filename):
                                             duration=6*3600,
                                             project=project)
 
+    def call_cmd(cmd, retry=True):
+        try:
+            if aria2c_exe is not None:
+                print("Downloading symbolic link with aria2c")
+            else:
+                print("Downloading symbolic link with wget")
+            subprocess.check_call(cmd, stderr=subprocess.STDOUT)
+        except subprocess.CalledProcessError as e:
+            msg = ""
+            if e and e.output:
+                msg = e.output.strip()
+            if e.returncode == 22 and retry:  # hotfix, DEVEX-1779
```

Comment:
In case the error persists we want to break from retries after a few retries, one example of retries with exponential backoff is [here](https://github.com/dnanexus/dx-toolkit/blob/c4906fe48f757260dd528efe9bf1d50f26076a46/src/python/dxpy/__init__.py#L782).

Reply:
Thanks